standard/standard_9.php

```diff
@@ -786,6 +789,22 @@ function array_combine(array $keys, array $values): array { }
  * @return bool true on success or false on failure.
  */
 #[Pure]
+function array_key_exists($key, $array): bool
+{ }
+
+#[PhpStormStubsElementAvailable('8.0')]
```

Comment:
I think `#[LanguageLevelTypeAware]` can be used here for the type of $array as well.

Reply:
done